src/client/components/IframeView.vue

```diff
@@ -12,6 +12,10 @@ const anchor = ref<HTMLDivElement>()
 const key = computed(() => props.src)
 const iframeEl = ref<HTMLIFrameElement>()
 const box = reactive(useElementBounding(anchor))
+const iframeLoaded = ref(false)
+
+if (props.src.startsWith('http'))
```

Comment:
Let's remove it so that it applies to all iframe (even if it's a non-Web request).